common/src/ipc/socket.rs

```diff
@@ -1,141 +1,133 @@
-use std::{path::PathBuf, time::Duration};
-
-use rustix::{
-    fd::OwnedFd,
-    net::{self, RecvFlags},
-};
-
-use super::Mmap;
-
-pub struct SocketMsg {
-    pub(super) code: u8,
-    pub(super) shm: Option<Mmap>,
+use std::env;
+use std::marker::PhantomData;
+use std::sync::OnceLock;
+use std::time::Duration;
+
+use rustix::fd::OwnedFd;
+use rustix::io::Errno;
+use rustix::net;
+
+use super::ErrnoExt;
+use super::IpcError;
+use super::IpcErrorKind;
```

Comment:
Is there a reason why you are doing imports like this? I don't particularly mind, but it isn't consistent with the rest of the codebase. I supposed we could normalize them later...

Reply:
I simply prefer this style. When I change imports, the resulting diff is on one line (with entire path). Also, I think it is easier to parse (entire path in one spot, I don't have to jump around to see who declared something like `fs` module) and edit in general (`dd` in vim).